Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1297,216 +1301,207 @@ void CFrame::PollHotkeys(wxTimerEvent& event)
 	{
 		HotkeyManagerEmu::GetStatus();
 		wxKeyEvent keyevent = 0;
-
-		if (IsHotkey(keyevent, HK_TOGGLE_THROTTLE))
-		{
-			Core::SetIsFramelimiterTempDisabled(false);
-		}
-		else
-		{
-			ParseHotkeys(keyevent);
-		}
+		ParseHotkeys(keyevent);
 	}
 }
 
 void CFrame::ParseHotkeys(wxKeyEvent &event)
 {
+	unsigned int i = NUM_HOTKEYS;
```

Comment:
shouldn't i be set to zero here, since it's just going to be reset in the loop header?